test/dfusionMultiSafes.js

```diff
@@ -29,22 +29,30 @@ const { checkCorrectnessOfDeposits } = require("../scripts/utils/price-utils")(w
 
 const { toErc20Units } = require("../scripts/utils/printing_tools")
 
+const TEN = new BN(10)
+
 const checkPricesOfBracketStrategy = async function(lowestLimit, highestLimit, bracketSafes, exchange) {
   const stepSizeAsMultiplier = Math.pow(highestLimit / lowestLimit, 1 / bracketSafes.length)
-  let multiplicator = new BN("10")
-  if ((lowestLimit + highestLimit) / 2 < 10) {
-    multiplicator = new BN("10000000")
-  }
+
   // Correctness assertions
   for (const [index, bracketAddress] of bracketSafes.entries()) {
+    let multiplicator = new BN("100")
+    if (lowestLimit * Math.pow(stepSizeAsMultiplier, index) < 10) {
+      multiplicator = new BN("1000000")
+    }
     const auctionElements = exchangeUtils.decodeOrdersBN(await exchange.getEncodedUserOrders(bracketAddress))
     assert.equal(auctionElements.length, 2)
     const [buyOrder, sellOrder] = auctionElements
+    const decimalsOfSellToken = await (await ERC20.at(await exchange.tokenIdToAddressMap.call(buyOrder.sellToken))).decimals()
+    const decimalsOfBuyToken = await (await ERC20.at(await exchange.tokenIdToAddressMap.call(buyOrder.buyToken))).decimals()
+
     // Check buy order prices
     assert.isBelow(
       Math.abs(
```

Comment:
ToDo: write a function measuring the difference between two BN. Then we can assert than the distance is smaller than 1 and we can set a high `multiplicator = new BN("1000000")` for all cases.